ext/mysqli/mysqli.stub.php

```diff
@@ -405,13 +405,13 @@ public function attr_set(int $attribute, int $value) {}
      * @return bool
      * @alias mysqli_stmt_bind_param
      */
-    public function bind_param(string $types, mixed &...$vars) {}
+    public function bind_param(string $types, mixed &$var, mixed &...$vars) {}
 
     /**
      * @return bool
      * @alias mysqli_stmt_bind_result
      */
-    public function bind_result(mixed &...$vars) {}
+    public function bind_result( mixed &$var, mixed &...$vars) {}
```

Comment:
I'm not sure about these two changes. They are technically correct, but they also don't seem particularly useful. bind_param accepts zero vars, but fails because types must be non-empty. bind_result does only accept one or more, but I think making it zero or more would make more sense from a signature perspective (it would still have to match the field count, which usually can't be zero).